index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Crete-Round">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+    <!--Nav menu and the logo-->
+
+    <header>
+        <div class="logo">
+            <a href="#">Sami</a>
+        </div>
+        <nav>
+            <i class="fa fa-bars"></i>
+        </nav>
+    </header>
```

Comment:
- [ ] Kindly make sure to match the provided [Figma design ](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)as much as possible. Kindly adjust the margins/ padding left and right for this section so that it matches the project design.

**Your current design:**
![image](https://user-images.githubusercontent.com/86318284/176667793-68c17ce7-8e05-44cf-89a0-6b1803fe07d0.png)

**The desired Figma design:**
![image](https://user-images.githubusercontent.com/86318284/176667917-5d376ccc-9f09-462b-bf3d-402fb784ca55.png)